elements/lisk-validator/src/constants.ts

```diff
@@ -20,6 +20,6 @@ export const MAX_PUBLIC_KEY_LENGTH = 32;
 export const MAX_INT32 = 2147483647;
 export const MIN_INT32 = MAX_INT32 * -1;
 export const MAX_UINT32 = 4294967295;
-export const MAX_INT64 = BigInt('9223372036854775807');
-export const MIN_INT64 = MAX_INT64 * BigInt(-1) - BigInt(1);
+export const MAX_SINT64 = BigInt('9223372036854775807');
```

Comment:
For such larger numbers its easy to get confused. So better to add a comment here for reviewer as well. I appreciate If you add similar comment for other values as well. 

```suggestion
export const MAX_SINT64 = BigInt('9223372036854775807');  // 2 ^ (64 - 1) - 1
```